website/src/pages/api/new_task/[task_type].ts

```diff
@@ -37,7 +37,8 @@ const handler = async (req, res) => {
   });
 
   // Update the backend with our Task ID
-  await oasstApiClient.ackTask(task.id, registeredTask.id);
```

Comment:
Controversial proposal: What if we really delete this here and move the auto-ack to when we submit the answer?

Reply:
in that case, shouldn't the auto ack happen in the (real) backend?

I don't see the advantage of acking & submitting immediately.

TBH, I still don't fully understand why we ack in the first place.

Reply:
I agree the acking is a bit weird.  Let's discuss in the next call about how needed it really is.  For now let's pretend its required and just work around the API as is.